### PR TITLE
fathom 1.35.0 (new cask)

### PR DESCRIPTION
**Pull request by mcw0933**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

Adds Fathom AI Notetaker as a cask

### CODE REVIEW

File: Casks/f/fathom.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  url "https://storage.googleapis.com/electron_releases/v#{version}/Fathom-darwin-arm64-#{version}.dmg",
+      verified: "storage.googleapis.com/electron_releases/"
+  name "Fathom"
+  desc "Video conference support, automatic transcription"
+  homepage "https://fathom.video/"
+
+  livecheck do
+    url "https://electron-update.fathom.video/"
+    regex(%r{<div id="version">v(\d+(?:\.\d+)*)</div>}i)
+  end
```
Comment (bevanjkay) [inline Casks/f/fathom.rb:14]:
```suggestion
  livecheck do
    url "https://electron-update.fathom.video/update/darwin_arm64/0.0.0"
    strategy :json do |json|
      json["name"]&.tr("v", "")
    end
  end

File: Casks/f/fathom.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+cask "fathom" do
+  version "1.35.0"
+  sha256 "5d6d923c32c89fb5de65e2d89eeae7542c90e9b8ae3e40489550092851b44f7e"
+
+  url "https://storage.googleapis.com/electron_releases/v#{version}/Fathom-darwin-arm64-#{version}.dmg",
+      verified: "storage.googleapis.com/electron_releases/"
+  name "Fathom"
+  desc "Video conference support, automatic transcription"
```
Comment (bevanjkay) [inline Casks/f/fathom.rb:8]:
```suggestion
  desc "Record and transcribe video conferences"

File: Casks/f/fathom.rb (hunk 1 above)
Comment (mcw0933) [reply Casks/f/fathom.rb]: Thank you!  I had been looking at trying the `yml` strategy here instead, didn't know this was an alternative.

File: Casks/f/fathom.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "Fathom Helper",
+    "Fathom",
+  ]
+  uninstall delete: [
+    "/Applications/Fathom.app",
+    "/Applications/Fathom.app/Contents/Frameworks/Fathom Helper (GPU).app",
+    "/Applications/Fathom.app/Contents/Frameworks/Fathom Helper (Plugin).app",
+    "/Applications/Fathom.app/Contents/Frameworks/Fathom Helper (Renderer).app",
+    "/Applications/Fathom.app/Contents/Frameworks/Fathom Helper.app",
+  ]
```
[the remark this answers is not in the document]
Comment (mcw0933) [reply Casks/f/fathom.rb:32]: sg - I was going off another cask as an example.

**Comment by mcw0933** [comment]
Seems I'm impacted by  #191585 (along with others).  Anything else I can do here, or just sit tight?

**Comment by khipp** [comment]
@mcw0933 Those CI failures are non-blocking, and we will merge the PR once it's approved (see [comment](https://github.com/Homebrew/homebrew-cask/issues/191585#issuecomment-2467775092)).

However, it appears that your last push reverted @bevanjkay's suggestions: https://github.com/Homebrew/homebrew-cask/compare/b7dae2552c20625956d53ebfae1b6deaede3ec6b..97db2c2446884e308ce39d0c040d8bbf49d64367

**Comment by mcw0933** [comment]
> However, it appears that your last push reverted @bevanjkay's suggestions

Ooops - yep, didn't pull those before rebasing.  I'll fix it, thanks

**Comment by khipp** [comment]
You'll need to rebase against `origin/master` to get the CI green again. The commit style check doesn't allow merging the master branch into PRs.

**Review by khipp — APPROVED** [review]
Thank you for your first contribution to `homebrew-cask`, @mcw0933! 🎉

**Pull request merged**